src/v3/src/transformer/button/getButtonControls.ts

```diff
@@ -70,13 +71,10 @@ export const getButtonControls = (
     ?? getButtonStep('currentAuthenticatorEnrollment-recover');
   if (config.stepWithForgotPassword && forgotPasswordStep) {
     const { name } = forgotPasswordStep;
-    const forgotPassword: ButtonElement = {
-      type: 'Button',
-      label: loc('forgotpassword', 'login'),
+    const forgotPassword: LinkElement = {
+      type: 'Link',
```

Comment:
Wonder if we should rename this transformer or move these to another new transformer

Reply:
on the list, will be in next few PRs